libs/ui/login-form/schematics/ng-add/index.ts

```diff
@@ -38,7 +38,7 @@ export const ngAdd = () => (tree: Tree, context: SchematicContext): Tree => {
     const nodeDependency: NodeDependency = {
       type: NodeDependencyType.Default,
       name: individualPackage[0],
-      version: individualPackage[1],
+      version: individualPackage[1].trim().trim(),
```

Comment:
I think this may be an extra paste..

Reply:
🤦 I tried one and then used `sed` to do the rest, but forgot to revert the first testing case..
good catch!